packages/client/bin/utils.ts

```diff
@@ -656,7 +656,7 @@ export async function generateClientConfig(args: ClientOpts) {
         // WASM errors with `unreachable` if we try to pass in less than 32 bytes in the message
         throw new Error('message length must be 32 bytes or greater')
       }
-      const buf = secp256k1Sign(msg, pk)
+      const buf = secp256k1Sign(msg, pk) // TODO: client defaults this to WASM: how do we handle this?
```

Comment:
This shouldn't matter.  The client is never "signing" anything as we don't use private keys in the client anywhere (except in `dev` mode and that should never be used with a real account's private key anyway)

Reply:
This is true. I checked though, the WASM does not have support for these hedged signatures.

Reply:
Here again, I don't think it matters.  If anything, we could update the docs for `customCrypto` to note that we don't suggest using `wasm-crypto` where you'll be managing private keys and signing stuff.  The `wasm-crypto` library can still recover addresses from these signatures so we're not going to break anything.

Reply:
I find it interesting that we inject the whole wasm package as custom crypto. You are right, it does not look like we are using signing anywhere in the client (except I think on PoA networks where we have to sign Clique). I think for completeness we just injected all WASM (so fast (?)) crypto's in custom crypto? 

I should have removed the TODO comment after I replied to this one, will do. You are right that when we are not signing anything from the client it does not matter (yet).